File: silly_print/ruby/lib/plugins/elephant.rb
module SillyPrint
  module Printers
    class Elephant < Base

      def format(message)
        <<ASCII
#{message}
     \\     /\\  ___  /\\
      \\   // \\/   \\/ \\\\
         ((    O O    ))
          \\\\ /     \\ //
           \\/  | |  \\/
            |  | |  |
            |  | |  |
            |   o   |
            | |   | |
            |m|   |m|

ASCII
      end

    end
  end
end

File: silly_print/php/lib/plugins/Elephant.php
<?php

namespace SillyPrint\Printers;

class Elephant extends Base
{
	function format($message)
	{
		return <<<ASCII
$message
     \     /\  ___  /\
      \   // \/   \/ \\
         ((    O O    ))
          \\ /     \ //
           \/  | |  \/
            |  | |  |
            |  | |  |
            |   o   |
            | |   | |
            |m|   |m|

ASCII;
	}
}
